jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -2043,7 +2047,7 @@ ecma_builtin_array_prototype_object_find (ecma_value_t predicate, /**< callback
       if (ecma_op_to_boolean (call_value))
       {
         /* 8.f */
-        ret_value = ecma_copy_value (get_value);
+        ret_value = is_find ? ecma_copy_value (get_value) : current_index;
```

Comment:
On line 2064, the return value must be `-1` for findIndex as well.

Reply:
Also please add a test case for it.

Reply:
Added the `-1` return value to line 2064 and modified the tests accordingly. :)